desktop/apps/artwork/components/artists/templates/biography.jade

```diff
@@ -15,9 +15,3 @@
     p= artist.bio.replace('born', 'b.')
 
 div(id="market-insights-container-" + artist._id)
-
-unless sd.ARTIST_MARKET_DATA === 'experiment'
```

Comment:
This removes the highlights/exhibition information that we used to display (now replaced by market insights).

It's now located in its own tab.